test/uk/gov/hmrc/merchandiseinbaggage/smoketests/AdditionalDeclarationImportSpec.scala

```diff
@@ -16,7 +16,7 @@
 
 package uk.gov.hmrc.merchandiseinbaggage.smoketests
 
-import uk.gov.hmrc.merchandiseinbaggage.model.api.{DeclarationId, DeclarationType, SessionId}
+import uk.gov.hmrc.merchandiseinbaggage.model.api.{DeclarationType, SessionId}
```

Comment:
As above, only an import change, is it needed?

Reply:
compilation warns